### DISCUSSION TITLE
Geographically-distributed S3 bucket

### DISCUSSION THREAD

**Discussion by psergiu** [Seagate/cortx#370] [General] [answered]
Hi,

Does Cortx supports Geographically-distributed S3 buckets ? I we had Lyvedrive racks in two or three separate datacenters (in different timezones), can we have a single "stretched" S3 bucket that can be written to and read from any of the locations locally ? Will this also work with the community edition ?

Thank you !

**Comment by gregnsk** [comment, marked as answer]
Not yet. It's a roadmap item
